Makefile

```diff
@@ -56,7 +56,7 @@ non-production-build: module-check ## Build the non-production site, which adds
 	GOMAXPROCS=1 hugo --cleanDestinationDir --enableGitInfo --environment nonprod
 
 serve: module-check ## Boot the development server.
-	hugo server --buildFuture --environment development
+	hugo serve --minify --buildFuture --environment development --disableFastRender --renderToMemory
```

Comment:
The change to memory rendering is probably OK, but we should check it works on svelte PCs, and I think it belongs in a separate PR.

Reply:
Agreed. Change to `serve` target reverted.